jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc

```diff
@@ -1,6 +1,6 @@
 ﻿////
 *******************************************************************
-* Copyright (c) 2019 Eclipse Foundation
+* Copyright (c) 2019, 2024 Eclipse Foundation
```

Comment:
Please remove this. It is not needed according to the EMO, and adding it adds unnecessary work in the future.

Reply:
Serves me right for assuming the copyright practices and restrictions would be universal.   I will certainly conform to the EMO (https://www.eclipse.org/projects/handbook/#ip-copyright-headers).  Thanks for pointing this out.

Reply:
From the same document:
> An Eclipse Foundation open source project is not a legal entity. Further, the Eclipse Foundation does not assume ownership of contributions (the Eclipse Foundation is only a copyright holder in those cases where an employee of the Eclipse Foundation has authored the contribution).

I understand it the Eclipse Foundation should not be the copyright holder, rather `Contributors to the Eclipse Foundation` should have been used.